docs/usage/ippool-affinity-node.md

```diff
@@ -1,96 +1,63 @@
 # SpiderIPPool node affinity
 
-*Spiderpool supports multiple ways to select ippool. Pod will select a specific ippool to allocate IP according to the corresponding rules that with different priorities. Meanwhile, ippool can use selector to filter its user.*
+*Spiderpool supports configuring the affinity between IP pools and Nodes, which enables only Pods running on specific Nodes to use the IP pools that be affinity to these Nodes. Node affinity should be regarded as a **filtering mechanism** rather than a [pool selection rule](TODO).*
```

Comment:
```suggestion
*Spiderpool supports for configuring the affinity between IP pools and Nodes. It means a pod running on a specific node can only use the IP pools that you have configured the affinity to the node. Node affinity should be regarded as a **filtering mechanism** rather than a [pool selection rule](TODO).*
```

Reply:
海峰老师，这个 PR 其实还没写完，Labels 里带了 `not-ready` 的 PR 其实不太需要提前 review。上面这段引言其实是全部不要的，我只是随便拷了段以前不要的东西，先填着充个数。😂

Reply:
并不是 "运行在特定 Node 的 Pod 只能使用 xxx IP 池" 而是 ”仅有运行在特定 Node 的 Pod 才能使用 xxx IP 池“，其实那些 Node 上的 Pod 还可以使用其他 IP 池的。

Reply:
ok :) 主要是没看懂这句话，哈哈 😄